menu.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
```

Comment:
I think this file was removed and refactored under antora-camel-ui, I don't think this change was intentional